test/functional/internal/test_utils.py

```diff
@@ -15,61 +15,75 @@
 import copy
 
 import pytest
+from mock import Mock
 
 from dynamodb_encryption_sdk.encrypted import CryptoConfig
 from dynamodb_encryption_sdk.identifiers import CryptoAction
 from dynamodb_encryption_sdk.internal.utils import encrypt_batch_write_item
 from dynamodb_encryption_sdk.material_providers import CryptographicMaterialsProvider
 from dynamodb_encryption_sdk.structures import AttributeActions, EncryptionContext
-from dynamodb_encryption_sdk.transform import ddb_to_dict
+from dynamodb_encryption_sdk.transform import dict_to_ddb
 
-from ..functional_test_vector_generators import attribute_test_vectors
+from ..functional_test_utils import diverse_item
 
 
-def get_test_item(standard_dict_format, partition_key, sort_key):
-    attributes = attribute_test_vectors("serialize")
+def get_test_item(standard_dict_format, partition_key, sort_key=None):
+    attributes = diverse_item()
 
     attributes = {"attr_" + str(pos): attribute[0] for pos, attribute in enumerate(attributes)}
```

Comment:
What was the intention of retaining this step? I thought it was just unpacking the test vectors when you had been using `attribute_test_vectors` as a source.

```python
>>> pprint(attributes)
{'binary': b'this is a bytestring! \x01',
 'binary_set': {b'\x00\x00\x00', b'\x00\x01\x00', b'\x00\x00\x02'},
 'decimal': Decimal('123.456'),
 'int': 5,
 'number_set': {3, 4, 5},
 'string': 'this is a string',
 'string_set': {'def', 'abc', 'geh'}}
>>> pprint({"attr_" + str(pos): attribute[0] for pos, attribute in enumerate(attributes)})
{'attr_0': 'i',
 'attr_1': 'd',
 'attr_2': 's',
 'attr_3': 'b',
 'attr_4': 'n',
 'attr_5': 's',
 'attr_6': 'b'}
```

Reply:
yup 100% that's a fail on my part, it needs to go.